zipkin-server/pom.xml

```diff
@@ -134,6 +134,18 @@
       <artifactId>jul-to-slf4j</artifactId>
     </dependency>
 
+    <!-- the "exec" jar will exclude these -->
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-slf4j</artifactId>
```

Comment:
https://github.com/spring-projects/spring-boot/tree/master/spring-boot-project/spring-boot-starters does not list this as a starter, confused where this comes from ?

Reply:
sorry I will fix this